QuoteEngine.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import yaml
+import random
+
+
+class QuoteEngine:
+
+    def __init__(self, logger):
+        self.quote_url = "https://cdn.rawgit.com/tulul/tulul-quotes/master/quote.yaml"  # noqa
```

Comment:
How come a single quote is one fewer keystroke than a double quote?

But I agree with single quotes anyway.